recipes/orjson/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+# Fun times -- by default, Rust/Cargo tries to link executables on Linux by
+# invoking `cc`. An executable of this name is not necessarily available. By
+# setting a magic environment variable, we can override this default.
+
+if [ "$c_compiler" = gcc ] ; then
+    case "$BUILD" in
+        x86_64-*) rust_env_arch=X86_64_UNKNOWN_LINUX_GNU ;;
+        aarch64-*) rust_env_arch=AARCH64_UNKNOWN_LINUX_GNU ;;
+        powerpc64le-*) rust_env_arch=POWERPC64LE_UNKNOWN_LINUX_GNU ;;
+        *) echo "unknown BUILD $BUILD" ; exit 1 ;;
+    esac
+
+    export CARGO_TARGET_${rust_env_arch}_LINKER=\$CC
```

Comment:
My bad, we don't need to escape `$` here.

Reply:
Talking about line 15, right?

Reply:
No, not this. This is the end of `case`, we need that.

I meant, on line 16, we need to replace `\$CC` with `$CC`

Reply:
That makes much more sense. Updated